src/main/python/image_processing/sample_extract.py

```diff
@@ -25,6 +38,9 @@ def extract(self, img):
             id_details: JSON obj (The extracted information)
         Todo:
         """
+        if 'remove_face' in self.preferences:
+            remove_face = self.preferences['remove_face'] == 'true'
```

Comment:
Why compare it to 'true'? Is it not already a boolean when posted via the AJAX request?